### ISSUE TITLE
New submissions for Friday, 20 September 2024 (showing 37 of 37 entries )

### ISSUE THREAD

**Issue by terrryahn78** [terrryahn78/daily_arxiv_noti#27]
## Emergence of coupling induced transparency by tuning purely dissipative couplings
- **Keywords:** ['hybrid quantum']
 - **Authors:** Kuldeep Kumar Shrivastava, Moulik Deviprasad Ketkar, Biswanath Bhoi, Rajeev Singh
 - **Subjects:** Subjects:
Quantum Physics (quant-ph)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2409.12577
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2409.12577
 - **Abstract**
 Controlled transitions between coupling induced transparency (CIT) and coupling induced absorption (CIA) are effects of both fundamental importance as well as potential applications in various devices. We have explored these peculiar phenomena in multi-mode coupled hybrid quantum systems by considering a tunable mode (TM) and several static modes (SMs). The individual SMs and TM are designed such that they show CIA, but upon coupling different SMs we observe a transition from CIA to CIT. Quite remarkably we are able to achieve CIT using only purely dissipative couplings, whereas it is well known that CIT appears with coherent coupling. We have developed a robust quantum theory based formalism which is able to capture the transition between CIA to CIT and have the capability to explain the inter-transition (CIT to CIA) as well as intra-transitions (CIA to CIA, CIT to CIT etc.) in a multimode hybrid quantum system all with just linear approach. A general model is developed for hybrid quantum systems having N modes. We have then explicitly described two sets of hybrid systems, the first set is of three modes, 1TM coupled with 2SMs, and the second set is of four modes, 1TM coupled with 3SMs. Later we have generalised it for hybrid quantum systems having N number of modes. The results provide a pathway for designing hybrid systems that can control the group velocity of light, offering potential applications in the fields of optical switching and quantum information technology. Our finding and formulation that in a single hybrid quantum system we can achieve controllable inter-transitions and intra-transitions of CIT/ CIA may open a tool and guidance for its application in quantum technology and quantum materials as the TMs/SMs may be well extended to other real/ quasi-particles also.
## Ultra-high strained diamond spin register with coherent optical link
- **Keywords:** ['hybrid quantum']
 - **Authors:** Marco Klotz, Andreas Tangemann, Alexander Kubanek
 - **Subjects:** Subjects:
Quantum Physics (quant-ph)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2409.12645
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2409.12645
 - **Abstract**
 Solid-state spin defects, such as color centers in diamond, are among the most promising candidates for scalable and integrated quantum technologies. In particular, the good optical properties of silicon-vacancy centers in diamond combined with naturally occurring and exceptionally coherent nuclear spins serve as a building block for quantum networking applications. Here, we show that leveraging an ultra-high strained silicon-vacancy center inside a nanodiamond allows us to coherently and efficiently control its electron spin, while mitigating phonon-induced dephasing at liquid helium temperature. Moreover, we indirectly control and characterize a 13C nuclear spin and establish a quantum register. We overcome limited nuclear spin initialization by implementing single-shot nuclear spin readout. Lastly, we demonstrate coherent optical control with GHz rates, thus connecting the register to the optical domain. Our work paves the way for future integration of quantum network registers into conventional, well-established photonics and hybrid quantum communication systems.
## Tight and Efficient Gradient Bounds for Parameterized Quantum Circuits
- **Keywords:** ['hybrid quantum']
 - **Authors:** Alistair Letcher, Stefan Woerner, Christa Zoufal
 - **Subjects:** Subjects:
Quantum Physics (quant-ph)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2309.12681
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2309.12681
 - **Abstract**
 The training of a parameterized model largely depends on the landscape of the underlying loss function. In particular, vanishing gradients are a central bottleneck in the scalability of variational quantum algorithms (VQAs), and are known to arise in various ways. However, a caveat of most existing gradient bound results is the requirement of t-design circuit assumptions that are typically not satisfied in practice. In this work, we loosen these assumptions altogether and derive tight upper and lower bounds on loss and gradient concentration for a large class of parameterized quantum circuits and arbitrary observables, which are significantly stronger than prior work. Moreover, we show that these bounds, as well as the variance of the loss itself, can be estimated efficiently and classically-providing practical tools to study the loss landscapes of VQA models, including verifying whether or not a circuit/observable induces barren plateaus. In particular, our results can readily be leveraged to rule out barren plateaus for a realistic class of ansätze and mixed observables, namely, observables containing a non-vanishing local term. This insight has direct implications for hybrid Quantum Generative Adversarial Networks (qGANs). We prove that designing the discriminator appropriately leads to 1-local weights that stay constant in the number of qubits, regardless of discriminator depth. This implies that qGANs with appropriately chosen generators do not suffer from barren plateaus even at scale-making them a promising candidate for applications in generative quantum machine learning. We demonstrate this result by training a qGAN to learn a 2D mixture of Gaussian distributions with up to 16 qubits, and provide numerical evidence that global contributions to the gradient, while initially exponentially small, may kick in substantially over the course of training.
## Investigating the Effect of Noise on the Training Performance of Hybrid Quantum Neural Networks
- **Keywords:** ['hybrid quantum']
 - **Authors:** Muhammad Kashif, Emman Sychiuco, Muhammad Shafique
 - **Subjects:** Subjects:
Quantum Physics (quant-ph)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2402.08523
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2402.08523
 - **Abstract**
 In this paper, we conduct a comprehensively analyze the influence of different quantum noise gates, including Phase Flip, Bit Flip, Phase Damping, Amplitude Damping, and the Depolarizing Channel, on the performance of HyQNNs. Our results reveal distinct and significant effects on HyQNNs training and validation accuracies across different probabilities of noise. For instance, the Phase Flip gate introduces phase errors, and we observe that HyQNNs exhibit resilience at higher probability (p = 1.0), adapting effectively to consistent noise patterns, whereas at intermediate probabilities, the performance declines. Bit Flip errors, represented by the PauliX gate, impact HyQNNs in a similar way to that Phase Flip error gate. The HyQNNs, can adapt such kind of errors at maximum probability (p = 1.0). Unlike Phase and Bit Flip error gates, Phase Damping and Amplitude Damping gates disrupt quantum information, with HyQNNs demonstrating resilience at lower probabilities but facing challenges at higher probabilities. Amplitude Damping error gate, in particular, poses efficiency and accuracy issues at higher probabilities however with lowest probability (p = 0.1),it has the least effect and the HyQNNs, however not very effectively, but still tends to learn. The Depolarizing Channel proves most detrimental to HyQNNs performance, with limited or no training improvements. There was no training potential observed regardless of the probability of this noise gate. These findings underscore the critical need for advanced quantum error mitigation and resilience strategies in the design and training of HyQNNs, especially in environments prone to depolarizing noise. This paper quantitatively investigate that understanding the impact of quantum noise gates is essential for harnessing the full potential of quantum computing in practical applications.
## Utilizing small quantum computers for machine learning and ground state energy approximation
- **Keywords:** ['hybrid quantum']
 - **Authors:** Stian Bilek
 - **Subjects:** Subjects:
Quantum Physics (quant-ph)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2403.14406
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2403.14406
 - **Abstract**
 Quantum circuit partitioning (QCP) is a hybrid quantum-classical approach that aims to simulate large quantum systems on smaller quantum computers. A quantum computation is divided into smaller subsystems and results of measurements on these subsystems are combined using classical processing. In this paper, we propose a QCP strategy to measure an observable on a large quantum system by utilizing several quantum systems of smaller size. The method can be applied to both machine learning and variational ground state energy approximation, and we show that the required calculations and the variance of the gradients can be tailored to scale efficiently with the total number of qubits. Thus it can be utilized to mitigate the well-known problem of barren plateaus. Additionally, the method can be realized by performing simple measurements of Pauli-strings on the separate subsystems, and the gradients can be estimated with common methods such as the parameter-shift rule. We demonstrate the method by approximating the ground state energy of the 1D transverse-field Ising model with periodic boundary conditions, and by classifying handwritten digits. For the ground state energy approximation, we achieved a relative error within the order of 0.1\% for all the tested system sizes. When applied to the classification between the digits 3 and 6, we were able to generalize to out-of-sample data with 100\% accuracy.
## CompressedMediQ: Hybrid Quantum Machine Learning Pipeline for High-Dimensional Neuroimaging Data
- **Keywords:** ['hybrid quantum']
 - **Authors:** Kuan-Cheng Chen, Yi-Tien Li, Tai-Yu Li, Chen-Yu Liu, Cheng-Yu Chen
 - **Subjects:** Subjects:
Quantum Physics (quant-ph); Distributed, Parallel, and Cluster Computing (cs.DC); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2409.08584
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2409.08584
 - **Abstract**
 This paper introduces CompressedMediQ, a novel hybrid quantum-classical machine learning pipeline specifically developed to address the computational challenges associated with high-dimensional multi-class neuroimaging data analysis. Standard neuroimaging datasets, such as 4D MRI data from the Alzheimer's Disease Neuroimaging Initiative (ADNI) and Neuroimaging in Frontotemporal Dementia (NIFD), present significant hurdles due to their vast size and complexity. CompressedMediQ integrates classical high-performance computing (HPC) nodes for advanced MRI pre-processing and Convolutional Neural Network (CNN)-PCA-based feature extraction and reduction, addressing the limited-qubit availability for quantum data encoding in the NISQ (Noisy Intermediate-Scale Quantum) era. This is followed by Quantum Support Vector Machine (QSVM) classification. By utilizing quantum kernel methods, the pipeline optimizes feature mapping and classification, enhancing data separability and outperforming traditional neuroimaging analysis techniques. Experimental results highlight the pipeline's superior accuracy in dementia staging, validating the practical use of quantum machine learning in clinical diagnostics. Despite the limitations of NISQ devices, this proof-of-concept demonstrates the transformative potential of quantum-enhanced learning, paving the way for scalable and precise diagnostic tools in healthcare and signal processing.
## Hybrid quantum cycle generative adversarial network for small molecule generation
- **Keywords:** ['hybrid quantum']
 - **Authors:** Matvei Anoshin, Asel Sagingalieva, Christopher Mansell, Dmitry Zhiganov, Vishal Shete, Markus Pflitsch, Alexey Melnikov
 - **Subjects:** Subjects:
Biomolecules (q-bio.BM); Emerging Technologies (cs.ET); Machine Learning (cs.LG); Biological Physics (physics.bio-ph); Quantum Physics (quant-ph)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2402.00014
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2402.00014
 - **Abstract**
 The drug design process currently requires considerable time and resources to develop each new compound that enters the market. This work develops an application of hybrid quantum generative models based on the integration of parametrized quantum circuits into known molecular generative adversarial networks, and proposes quantum cycle architectures that improve model performance and stability during training. Through extensive experimentation on benchmark drug design datasets, QM9 and PC9, the introduced models are shown to outperform the previously achieved scores. Most prominently, the new scores indicate an increase of up to 30% in the quantitative estimation of druglikeness. The new hybrid quantum machine learning algorithms, as well as the achieved scores of pharmacokinetic properties, contribute to the development of fast and accurate drug discovery processes.
